### DISCUSSION TITLE
Override the appendonly no

### DISCUSSION THREAD

**Discussion by yaztriumph** [OT-CONTAINER-KIT/redis-operator#343] [Q&A]
Hi all, can we override the redis.conf with redis-external-config configmap appendonly no? I've applied but didn't work. Any suggestions, thank you!
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: redis-external-config
data:
  redis-external.conf: |
    tcp-keepalive 400
    slowlog-max-len 158 
    stream-node-max-bytes 2048
    appendonly no
```

**Comment by enenum** [comment]
me too. anyone?

**Comment by alita1991** [comment]
I have the same issue, I can't restore from backup on standalone, because i can't disable AOF

**Comment by drivebyer** [comment]
What image version do you use?
